src/components/chapters/ChapterHeader/ChapterHeader.module.scss

```diff
@@ -1,17 +1,73 @@
 .container {
-  text-align: center;
+  display: flex;
+  justify-content: space-between;
   padding: var(--spacing-xxsmall) 0;
   margin: var(--spacing-xxsmall) 0;
+  position: relative;
+  direction: ltr;
```

Comment:
Is there a specific reason we need to specify this?

Reply:
yeah, in reading view the direction of this component's parent is`rtl`. 

and It affects the flex-direction in this component, which doesn't match the spec. So we need rtl